packages/preferences/src/browser/views/components/preference-markdown-renderer.ts

```diff
@@ -47,19 +52,26 @@ export class PreferenceMarkdownRenderer {
         engine.renderer.rules.code_inline = (tokens, idx, options, env, self) => {
             const token = tokens[idx];
             const content = token.content;
-            if (content.startsWith('#') && content.endsWith('#')) {
-                const preferenceId = content.substring(1, content.length - 1);
-                const preferenceNode = this.model.getNodeFromPreferenceId(preferenceId);
+            if (content.length > 2 && content.startsWith('#') && content.endsWith('#')) {
+                const id = content.substring(1, content.length - 1);
+                // First check whether there's a preference with the given ID
+                const preferenceNode = this.model.getNodeFromPreferenceId(id);
                 if (preferenceNode) {
                     let name = this.labelProvider.getName(preferenceNode);
                     const prefix = this.labelProvider.getPrefix(preferenceNode, true);
                     if (prefix) {
                         name = prefix + name;
                     }
-                    return `<a title="${preferenceId}" href="preference:${preferenceId}">${name}</a>`;
-                } else {
-                    console.warn(`Linked preference "${preferenceId}" not found.`);
+                    return `<a title="${id}" href="preference:${id}">${name}</a>`;
                 }
+                // If no preference was found, check whether there's a command with the given ID
+                const command = this.commandRegistry.getCommand(id);
+                if (command) {
+                    const name = `${command.category ? `${command.category}: ` : ''}${command.label}`;
+                    return `<a class="disabled-link" title="${id}">${name}</a>`;
```

Comment:
I'm wondering why we're using and anchor tag here at all: if the idea is just to style the text, why not use a `<div>` and add css to make it look right?

Reply:
You're right, it's now using a `span` instead.